apps/www/registry/default/example/combobox-form.tsx

```diff
@@ -100,7 +100,7 @@ export default function ComboboxForm() {
                     <CommandGroup>
                       {languages.map((language) => (
                         <CommandItem
-                          value={language.label}
+                          value={language.value}
```

Comment:
Changing to `language.value` will make the search by `language.key` which is against what we want to achieve in this example. You are facing this issue because you have duplicate labels.